packages/mermaid/src/diagrams/pie/pieRenderer.js

```diff
@@ -70,6 +71,17 @@ export const draw = (txt, id, _version, diagObj) => {
       sum += data[key];
     });
 
+    const legendShowData = diagObj.db.getShowData() || conf.showData || conf.pie.showData || false;
```

Comment:
We should use `??` instead of `||` and arrange the configs in the order of importance. Now if any one of these config is true, showData will be true.

Reply:
Well, as this is a renderer, it should only rely on `diagObj.db.getShowData()`